README.md

```diff
@@ -1,12 +1,23 @@
-# Reusable JupyterHub Pytest Plugin
+**[Description](#description)** |
+**[Installation](#installation)** |
+**[Usage](#usage)** |
+**[Contributing](#contributing)**
+
+---
+
+# [Reusable JupyterHub Pytest Plugin](https://github.com/jupyterhub/pytest-jupyterhub)
+
+[![Documentation status](https://img.shields.io/readthedocs/pytest-jupyterhub?logo=read-the-docs)](https://pytest-jupyterhub.readthedocs.io/en/latest/?badge=latest)
+[![GitHub Workflow Status - Test](https://github.com/jupyterhub/pytest-jupyterhub/actions/workflows/test.yml/badge.svg)](https://github.com/jupyterhub/pytest-jupyterhub/actions)
```

Comment:
I want us to consistently use the img.shields.io for consistent badges styling, the img.shields.io badges for github workflows needs an update in many of our github repos to work after a change though. Here is an updated version!

```suggestion
[![GitHub Workflow Status - Test](https://img.shields.io/github/actions/workflow/status/jupyterhub/pytest-jupyterhub/test.yml?logo=github&label=tests&branch=main)](https://github.com/jupyterhub/pytest-jupyterhub/actions)
```